locales/en/JussenButton.json

```diff
@@ -0,0 +1,3 @@
+{
+  "10sen-filter-button": "filtering to 10sen in {{year}}"
```

Comment:
"filtering to 10sen" should be "Filter to 10sen". Everything else looks OK!

Reply:
Thank you!